app/services/model/RejectionReason.scala

```diff
@@ -62,6 +64,9 @@ object RejectionReason {
     }.getOrElse(Seq.empty)
   }
 
+  private def logMissingPointerMessageKey(pointer: Option[Pointer])(implicit messages: Messages): Unit =
```

Comment:
```
error.pointer.filter { p =>
 val defined = messages.isDefinedAt(p.messageKey)
 if(!defined) {
    logger.warn("...")
 }
 defined
}
```

Reply:
Not much different but saves traversing the pointer list twice.

Up to you which you prefer

Reply:
Good point

Reply:
Fixed